app/views/movies/show.html.erb

```diff
@@ -1,62 +1,43 @@
-<section class="movie-details">
-   <div class="image">
-    <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
-      <div class="carousel-inner">
-        <% @movie.images.each_with_index do | item, index | %>
-          <div class="carousel-item <%= index == 0 ? 'active' : '' %> ">
-            <%= image_tag item, class: "d-block w-100" %>
-          </div>
-        <% end %>
-      </div>
-      <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev">
-        <span class="carousel-control-prev-icon" aria-hidden="true"></span>
-        <span class="sr-only">Previous</span>
-      </a>
-      <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next">
-        <span class="carousel-control-next-icon" aria-hidden="true"></span>
-        <span class="sr-only">Next</span>
-      </a>
+<section class="flex">
+<div>
+    <div class="h-120 p-8">
+      <% @movie.images.each_with_index do | item, index | %>
+        <%= image_tag item %>
+      <% end %>
     </div>
 
```

Comment:
Adding newline should be done following a project-wide convention or when the code would be less readable. This seems random.